### ISSUE TITLE
Possible glyph display or dictionary file issue.

### ISSUE THREAD

**Issue by terrywallwork** [mvysny/aedict#913]
Android ８ tablet paid version of Aedict latest version from Google play store.
I noticed a display issue for an entry in the dictionary.

牛� [ごこう]: bovine bile

It shows a diamond question mark.  see screen shot.

![Screenshot_1584121530777](https://user-images.githubusercontent.com/19387982/76646751-718d5480-6553-11ea-8d38-124901a4890e.jpg)

**Comment by mvysny** [comment]
You're right! Looks like a bug in the `buddhdic` source itself which also contains the following line:
```
牛� [ごこう] /bovine bile/
```

However, the data seems to originate from http://www.buddhism-dict.net/ddb/ where the search for "bovine bile" (use 'guest' with no password) will correctly yield 牛黃.

The DDB site doesn't provide access to the full XML directly. Quoting Jim Breen from ftp://ftp.net.usf.edu/pub/ftp.monash.edu.au/pub/nihongo/buddhdic.txt :

> This file is an extract from Chuck Muller's monumental
Digital Dictionary of Buddhism (DDB) (see http://www.acmuller.net/ddb/index.html)
>
> Chuck kindly provided an extract in XML format of the full database. From
this I generated an EDICT-format version containing the Japanese headword
(kanji and reading) and the short English meaning.

From that I judge that the DDB authors do not provide a free download of their data, and would probably not want their data to be automatically indexed by Aedict...? Let me think on the topic please.

**Comment by mvysny** [comment]
Regardless, this needs to be solved upstream. However, since http://www.acmuller.net/ddb/index.html is a 404 NOT FOUND now, perhaps the buddhdic is not active anymore?

Could you please contact the buddhdic authors in order to fix the bug in the dictionary file?
